src/function/itemLikes.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-return-await */
+/* eslint-disable no-console */
+const involvementLikesURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9YYfJY5gRMr3MgkvmC9i/Likes/';
+
+const postLike = async (itemId) => {
+  try {
+    await fetch(involvementLikesURL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        item_id: `item${itemId}`,
+      }),
+    });
+  } catch (error) {
+    console.error(error);
```

Comment:
- [ ] Please don't use `console` in your code because it's causing less optimization. 😄 🙏 
_Please see about this tip and other useful ones right [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.)._

Reply:
Issue done, thanks for your feedback 👏